pyiso/ercot.py

```diff
@@ -276,6 +280,8 @@ def get_lmp(self, node_id='HB_HUBAVG', **kwargs):
             report_name = 'rt5m_lmp'
         elif self.options['market'] == self.MARKET_CHOICES.dam:
             report_name = 'dam_hrly_lmp'
+        elif self.options['market'] == self.MARKET_CHOICES.allbusrt:
```

Comment:
You should put something like if ```node_id=='ALLBUS_RT': request_report('allbus_rt')``` and handle the special formatting of the report in a format_allbus_rt() function.